pkg/model/stack_serializer.go

```diff
@@ -41,24 +41,29 @@ type StackRaw struct {
 
 //Service represents an okteto stack service
 type ServiceRaw struct {
-	Deploy          *DeployInfoRaw     `yaml:"deploy,omitempty"`
-	Build           *BuildInfo         `yaml:"build,omitempty"`
-	CapAdd          []apiv1.Capability `yaml:"cap_add,omitempty"`
-	CapDrop         []apiv1.Capability `yaml:"cap_drop,omitempty"`
-	Command         Args               `yaml:"command,omitempty"`
-	Entrypoint      Command            `yaml:"entrypoint,omitempty"`
-	Args            Args               `yaml:"args,omitempty"`
-	EnvFiles        []string           `yaml:"env_file,omitempty"`
-	Environment     *RawMessage        `yaml:"environment,omitempty"`
-	Expose          *RawMessage        `yaml:"expose,omitempty"`
-	Image           string             `yaml:"image,omitempty"`
-	Labels          *RawMessage        `json:"labels,omitempty" yaml:"labels,omitempty"`
-	Annotations     map[string]string  `json:"annotations,omitempty" yaml:"annotations,omitempty"`
-	Ports           []PortRaw          `yaml:"ports,omitempty"`
-	Scale           int32              `yaml:"scale"`
-	StopGracePeriod *RawMessage        `yaml:"stop_grace_period,omitempty"`
-	Volumes         []StackVolume      `yaml:"volumes,omitempty"`
-	WorkingDir      string             `yaml:"working_dir,omitempty"`
+	Deploy                   *DeployInfoRaw     `yaml:"deploy,omitempty"`
+	Build                    *BuildInfo         `yaml:"build,omitempty"`
+	CapAdd                   []apiv1.Capability `yaml:"cap_add,omitempty"`
+	CapAddCamelCase          []apiv1.Capability `yaml:"capAdd,omitempty"`
```

Comment:
Since camel case is the official one, let' call them `CapAddSneakCase` and `CapAdd`. 
In the long long run, we might want to deprecate the sneak notation